Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -44,7 +44,23 @@ try {
   
     $containerName = GetContainerName($project)
 
-    $runAlPipelineParams = @{}
+    $ap = "$ENV:GITHUB_ACTION_PATH".Split('\')
+    $branch = $ap[$ap.Count-2]
+    $owner = $ap[$ap.Count-4]
+
+    if ($owner -ne "microsoft") {
+        $verstr = "dev"
+    }
+    else {
+        $verstr = $branch
+    }
+
+    $runAlPipelineParams = @{
+        "sourceRepositoryUrl" = "$ENV:GITHUB_SERVER_URL/$ENV:GITHUB_REPOSITORY"
+        "sourceCommit" = $ENV:GITHUB_SHA
+        "buildBy" = "AL-Go for GitHub,$verstr"
```

Comment:
Suggestion: "$owner/AL-Go/$branch"?

Reply:
The goal isn't to find the exact AL-Go version who created the app - it is just to identify product and version.
Build By will be
- BcContainerHelper,version (if people are calling Compile-AppInBcContainer manually)
- ALOps,version (if people are using ALOps)
- Alpaca,version (if people are using Alpaca)
- VSCode (if people are compiling directly from VSCode)